Plugins/Flow.Launcher.Plugin.WindowsSettings/Helper/ResultHelper.cs

```diff
@@ -54,7 +54,6 @@ internal static class ResultHelper
                     if (areaMatch.IsSearchPrecisionScoreMet())
                     {
                         var settingResult = NewSettingResult(areaMatch.Score + midScore, entry.Type);
-                        settingResult.SubTitleHighlightData = areaMatch.MatchData.Select(x => x + 6).ToList();
```

Comment:
why removed?

Reply:
because translation will make that behave very weird.